sytest/pipeline.yml

```diff
@@ -200,22 +200,130 @@ steps:
           image: "matrixdotorg/sytest-dendrite"
           propagate-environment: true
           always-pull: true
-          workdir: "/sytest"
+          workdir: "/src"
```

Comment:
I'm confused what's going on here. we run a command which attempts to construct `/src`. How does it work if we change to /src first?

Reply:
... and why are you making it out of sync with the synapse runs?

Reply:
Good question, I have no idea. Also I'm not sure what you mean "out of sync"? I've just copied and pasted from the Dendrite pipelines.

Reply:
I've put it back to `/sytest`.